packages/patternfly-4/react-core/src/experimental/components/Drawer/Drawer.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Drawer/drawer';
+import { css } from '@patternfly/react-styles';
+
+export interface DrawerProps extends React.HTMLProps<HTMLDivElement> {
+  /** Additional classes added to the Drawer. */
+  className?: string;
+  /** Content rendered inside the button */
+  children?: React.ReactNode;
+  /** Indicate if the drawer is expanded */
+  isExpanded: boolean;
+  /** Indicates if the content element and panel element are displayed side by side. */
+  isInline: boolean;
+  /** Content to rendered in the left hand panel */
```

Comment:
I think the comments are opposite between this and `children`
```suggestion
  /** Content rendered in the drawer */
```